diff -urN libnm-qt-0.9.8.3/CMakeLists.txt libnm-qt-0.9.8.3-patched/CMakeLists.txt
--- libnm-qt-0.9.8.3/CMakeLists.txt	2014-11-05 20:07:23.961613344 +1000
+++ libnm-qt-0.9.8.3-patched/CMakeLists.txt	2014-11-05 20:07:17.255613039 +1000
@@ -4,7 +4,7 @@
 
 include(GNUInstallDirs)
 
-set(VERSION 0.9.8.2)
+set(VERSION 0.9.8.3)
 set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_MODULE_PATH})
 
 if (NOT CMAKE_BUILD_TYPE)
